src/pages/index.js

```diff
@@ -1,24 +1,103 @@
-import React, { useContext, useEffect } from "react"
+import React, { useContext, useEffect, useState } from "react"
+import useInterval from "../hooks/useInterval"
```

Comment:
Codacy found an issue: ['useInterval' is defined but never used.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5837031)